components/http_foundation.rst

```diff
@@ -362,6 +362,23 @@ analysis purposes. Use the ``anonymize()`` method from the
     $anonymousIpv6 = IpUtils::anonymize($ipv6);
     // $anonymousIpv6 = '2a01:198:603:10::'
 
+If you need even more anonymization, you can use the second and third parameters
```

Comment:
Can you add all this as info to the Code PR ? Thanks

Reply:
Added :slightly_smiling_face: